tennis-scheduler-web-app/src/components/Navbar.jsx

```diff
@@ -12,14 +42,25 @@ export const NavbarStart = () => {
         </Link>
         <Navbar.Toggle aria-controls="basic-navbar-nav" />
         <Navbar.Collapse id="basic-navbar-nav">
-          <Nav className="me-auto">
-            <Link className="nav-link" to="/login">
-              Login
-            </Link>
-            <Link className="nav-link" to="/registration">
-              Register
-            </Link>
-          </Nav>
+          {(() => {
+            if (!admin && !tennisPlayer) {
+              return (
+                <Nav className="me-auto">
+                  <Link className="nav-link" to="/login">Login</Link>
+                  <Link className="nav-link" to="/registration">Register</Link>
+                </Nav>
+              )
+            } else {
+              return (
+                <>
+                  <Nav className="me-auto">
+                    <Link className="nav-link" to="/profile">Profile</Link>
+                  </Nav>
+                  <button className="logout-button" onClick={logout}>Log out</button>
+                </>
+              )
+            }
+          })()}
```

Comment:
What if admin have something that player don't, add if for different roles

Reply:
That is not the case here,  if it comes to that, we'll add. For now, I can add only empty fragments for that cases if that is what you suggest?

Reply:
Just don't forget to add it later if you need it